src/mcp2515.cpp

```diff
@@ -108,46 +108,8 @@ MCP2515::ERROR MCP2515::reset(void) {
 
   delay(10);
 
-  uint8_t zeros[14];
```

Comment:
I feel like this might be the scariest commit. Like the commit message says, I truly think most of this is not necessary considering they send the reset command to the chip on line 106.